docs/snippets/vue/storybook-preview-global-decorator.js.mdx

```diff
@@ -1,5 +1,5 @@
 ```js
 // .storybook/preview.js
 
-export const decorators = [() => ({ template: '<div style="margin: 3em;"><story /></div>' })];
+export const decorators = [(story) => ({ template: '<div style="margin: 3em;"><story /></div>' })];
```

Comment:
@CyberStrike you can rename the file to: "storybook-preview-with-styled-components-decorator.js.mdx" and add the following:
```js
// .storybook/preview.js
import { ThemeProvider } from "vue-styled-components";


export const decorators = [
  (story) => ({
    components: {
      "theme-provider": ThemeProvider,
    },
    template: `<theme-provider :theme="{
    primary: 'palevioletred',
    secondary: 'papayawhip',
    tertiary: 'lavenderblush'
  }"><story/></theme-provider>`,
  }),
];
```

I did a little bit of investigation and it seems that styled components are available for Vue. Which applied to this case we can maintain the same pattern. Pardon the bad indent.

Reply:
Got it will update.

Reply:
`storybook-preview-global-decorator.js.mdx` is being used on `writing-stories/decorators` so I've kept it and created a new `storybook-preview-with-styled-components-decorator.js.mdx`

A note on this section for Vue developers chances are they would run into issues with Vue Modules when setting up an existing project at this point. But additions is another scope.